packages/babel-preset-env/README.md

```diff
@@ -187,6 +187,12 @@ Example environments: `chrome`, `opera`, `edge`, `firefox`, `safari`, `ie`, `ios
 
 The [data](https://github.com/babel/packages/babel-preset-env/blob/master/data/plugins.json) for this is generated by running the [build-data script](https://github.com/babel/packages/babel-preset-env/blob/master/scripts/build-data.js) which pulls in data from [compat-table](https://kangax.github.io/compat-table).
 
+### `targets.ecmascript`
+
+` "5" | "2015" | "2016" | "2017" | "2018"`
```

Comment:
Since we support `shippedProposals`, should we add a stage3/shippedProposals option?

Reply:
I think the problem of that is it changes over time (especially for the published package use case). When targeting specific browsers or ECMAScript versions, the output would always be the same.

Reply:
@rtsao Makes sense 👍